### DISCUSSION TITLE
Sort seems to have no effect

### DISCUSSION THREAD

**Discussion by rsworden59** [community/community#24857] [Enterprise]
<p>curl -s <a href="https://github.company.com/api/v3/orgs/orgname/repos?sort=full_name" rel="nofollow noopener">https://github.company.com/api/v3/orgs/orgname/repos?sort=full_name</a> | grep full_name</p>
<p>The results are in the default order “created”, not the requested order “full_name”. The sort option seems to have no effect. Other options like per_page=100 or type=private seem to work, so I think my query syntax is OK. Am I doing something wrong? </p>

**Comment by lee-dohm** [comment, marked as answer]
<p>It looks like you’re running against a GitHub Enterprise instance? If you reference the <a href="https://developer.github.com/enterprise/2.16/v3/repos/#list-organization-repositories" rel="nofollow noopener">GitHub Enterprise API documentation</a>, it doesn’t appear that the <code>sort</code> parameter is supported currently.</p>
<p>I hope that helps! Let us know if you have any more questions.</p>

**Comment by Mizumaki** [comment]
<p>Hmm, I faced the same issue.</p>
<p>And find out under <a href="https://developer.github.com/enterprise/2.16/v3/repos/#list-organization-repositories" rel="nofollow noopener">v2.16</a>, <code>sort</code> is not supported, but over <a href="https://developer.github.com/enterprise/2.17/v3/repos/#list-organization-repositories" rel="nofollow noopener">v2.17</a>, it is supported.</p>
<p>hmmmmmmm :sob:</p>
